source/loop/statsd.rst

```diff
@@ -0,0 +1,36 @@
+======
+Statsd
+======
+
+Loop-Server have got a number of Statsd counters and timers that can
+help monitor what's going on in near real-time.
+
++-------------------------------------------------------+----------+------------------------------------------------------------------+
+| Name                                                  | Type     | Description                                                      |
++=======================================================+==========+==================================================================+
+| ``loop.activated-users``                              | counter  | Count new Hawk Session created for a user.                       |
```

Comment:
With the type column I think we can remove "Count" and "Time" from the description column to make it even more succint